src/gui/transferlistsortmodel.cpp

```diff
@@ -105,18 +107,58 @@ bool TransferListSortModel::lessThan(const QModelIndex &left, const QModelIndex
 {
     Q_ASSERT(left.column() == right.column());
 
-    const int sortColumn = left.column();
+    if (m_sortColumns.empty() || (m_sortColumns.front() != left.column()))
+        m_sortColumns.push_front(left.column());
```

Comment:
So it's not the same subsorting as it was before? It just remembers the previously used sorts and uses them as a fallback?

Reply:
> It just remembers the previously used sorts and uses them as a fallback?

yes